### DISCUSSION TITLE
Serious question about using MAC address

### DISCUSSION THREAD

**Discussion by PuckStar** [ESPresense/ESPresense#759] [Q&A]
I'm wondering why it's not possible to "see" my Samsung A50 phone using it's permanent MAC address just like [this tracker](https://github.com/andrewjfreyer/monitor) does.
That tracker sees my phone always with the same MAC and can track it like that.
Why is that not possible with this ESPresence and do I need to pair or use an app or whatever.

I'm honestly curious about this.

**Comment by DTTerastar** [comment]
I'd bet it'd be the difference between Bluetooth Classic and BTLE.  This project doesn't use any classic bluetooth.
